src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -283,12 +283,16 @@ public void visitToken(DetailAST ast) {
         final int lineNo = ast.getLineNo();
         final TextBlock cmt =
             contents.getJavadocBefore(lineNo);
-        if (cmt == null) {
+        if (cmt != null) {
+            checkTag(lineNo, cmt.getText());
+        }
+        else if (tag != null) {
             log(lineNo, MSG_MISSING_TAG, tag);
```

Comment:
this does not solve issue. There should be no message in case there is no javadoc, regardless tags and format